### PR TITLE
Support extra volumes

### PR DESCRIPTION
**Pull request by DXist**
#### What this PR does / why we need it:
This PR allows to provide extra volumes to mount in the pods. I use cert-manager to generate Rabbitmq certificate. And I mount 2 tls secrets using 2 volumes - server certificate and CA certificate.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by DXist** [comment]
@steven-sheehy , I've completed checklist items. 

Regarding `rabbitmqCert.existingSecret` - yes, I use it to provide server tls certificate. Extra volume is used to pass CA tls certificate.

**Comment by steven-sheehy** [comment]
> Regarding `rabbitmqCert.existingSecret` - yes, I use it to provide server tls certificate. Extra volume is used to pass CA tls certificate.

I'm still unclear about the use case. You can either supply `rabbitmqCert.cacertfile` contents to provide a CA tls certificate. Or you can use `rabbitmqCert.existingSecret` and supply the cacert.pem, cert.pem and key.pem all in the same secret similar to how the [existing secret](https://github.com/helm/charts/blob/76f08fe30c774d8ad76d8552ca38be5a370ed8f9/stable/rabbitmq-ha/templates/secret.yaml#L39) does. Then you can configure rabbit ssl options: 
```
ssl_options.cacertfile  = /etc/cert/cacert.pem
ssl_options.certfile = /etc/cert/cert.pem
ssl_options.keyfile = /etc/cert/key.pem
```

Are you saying you don't control the format of the server certificate and ca certificate and they have to be in separate secrets?

**Comment by DXist** [comment]
@steven-sheehy , I use [cert-manager](https://github.com/jetstack/cert-manager) to manage TLS certificates. They are created as TLS secrets on demand. Custom format will require extra step before `rabbitmq-ha` chart installation to create custom secret.

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.8
-version: 1.17.0
+version: 1.17.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: We bump minor versions for new features: 1.18.0

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (DXist) [reply stable/rabbitmq-ha/Chart.yaml]: Bumped version to 1.18.0

**Pull request merged**